docs/DEPLOYMENT.md

```diff
@@ -292,9 +291,7 @@ For `OpenShift`:-
 curl -k -X GET $(oc get -n tekton-hub routes api --template='https://{{ .spec.host }}/v1/categories')
 ```
 
-NOTE: At this moment, there are no resources in the db. Only the categories from hub config are added in the db.
-
-Let's deploy the UI first and then we will add the resources in db.
+NOTE: At this moment, resources are added in the db from the catalogs also categories from hub config are added in the db.
```

Comment:
do we need this `NOTE` ?

Reply:
I think we can remove it now cc: @PuneetPunamiya @piyush-garg